man/fi_endpoint.3.md

```diff
@@ -349,14 +349,14 @@ binding an endpoint to a counter, the following flags may be specified.
   initiated from the endpoint has completed successfully or in error.
```

Comment:
These flags are specifically documenting the use when binding an endpoint to a counter.  See grayed out section above line 349 above.  Generating an CQ entry is not formally defined actually.